src/styles/stopwatch.scss

```diff
@@ -0,0 +1,52 @@
+.stopwatch {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+
+  height: $stopwatch-size;
+  width: $stopwatch-size;
+  border: 4px solid rgb(42, 0, 119);
+  border-radius: 50%;
+
+  &__center {
+    position: relative;
+
+    display: block;
+    height: $center-size;
+    width: $center-size;
+    background-color: black;
+    border-radius: 50%;
+  }
+
+  &__min {
+    position: absolute;
+    bottom: 50%;
+    left: 7px;
+    z-index: -1;
+
+    display: block;
+    width: 14px;
+    height: 200px;
+    background-color: yellowgreen;
+    border: 1px solid rgb(17, 0, 26);
+
+    transform-origin: bottom center;
+    animation: spin steps(60) 60s infinite;
+  }
+
+  &__sec {
+    position: absolute;
+    bottom: 50%;
+    left: 7px;
+    z-index: -1;
+
+    display: block;
+    width: 5px;
+    height: 350px;
+    background-color: rgb(162, 226, 255);
+    border: 3px solid rgb(17, 0, 26);
```

Comment:
- color 'rgb(17, 0, 26)' is duplicated, so it makes sense to move it to variables
 - please check other duplicated styles and move them to variables